apps/ejabberd/src/mongoose_lib.erl

```diff
@@ -0,0 +1,11 @@
+-module(mongoose_lib).
```

Comment:
What is the purpose of this module? Do we really have to parse the argument to get an int value from it? Why simple binary_to_integer won't work here?